tests/src/podman-machine-onboarding.spec.ts

```diff
@@ -0,0 +1,197 @@
+        resourcesPage = new ResourcesPage(page);
+        const podmanResourcesSectionLocator: Locator = resourcesPage.getPage().getByLabel('podman', { exact: true });
+        await playExpect(podmanResourcesSectionLocator).toBeVisible();
+        await podmanResourcesSectionLocator.getByLabel('Setup Podman').click();
+        podmanOnboardingPage = await checkPodmanMachineOnboardingPage(page);
+      });
+    });
+    test.skip('Verify Podman Autostart is enabled and proceed to next page', async () => {
+      await playExpect(podmanOnboardingPage.podmanAutostartToggle).toBeChecked();
+      await podmanOnboardingPage.nextStepButton.click();
+    });
+    test.skip('Expect no machine created message and proceed to next page', async () => {
+      await playExpect(podmanOnboardingPage.onboardingStatusMessage).toHaveText(
+        `We could not find any Podman machine. Let's create one!`,
+      );
+      await podmanOnboardingPage.nextStepButton.click();
+    });
+    test.skip('Verify default podman machine settings', async () => {
+      await playExpect(podmanOnboardingPage.createMachinePageTitle).toHaveText(`Create a Podman machine`);
+      await playExpect(podmanOnboardingPage.podmanMachineConfiguration).toBeVisible();
+      await playExpect(podmanOnboardingPage.podmanMachineName).toHaveValue('podman-machine-default');
+      await playExpect(podmanOnboardingPage.podmanMachineCPUs).toBeVisible();
+      await playExpect(podmanOnboardingPage.podmanMachineMemory).toBeVisible();
+      await playExpect(podmanOnboardingPage.podmanMachineDiskSize).toBeVisible();
+      await playExpect(podmanOnboardingPage.podmanMachineImage).toHaveValue('');
+      await playExpect(podmanOnboardingPage.podmanMachineRootfulCheckbox).toBeChecked();
+      await playExpect(podmanOnboardingPage.podmanMachineUserModeNetworkingCheckbox).not.toBeChecked();
+      await playExpect(podmanOnboardingPage.podmanMachineStartAfterCreationCheckbox).toBeChecked();
+    });
+  });
+  describe.skip('Podman Machine creation and operations', async () => {
+    test('Create a default Podman machine', async () => {
+      const createMachineButton: Locator = podmanMachineConfiguration.getByRole('button', { name: 'Create' });
+      await createMachineButton.click();
+      const expandLogsButton: Locator = podmanOnboardingPage.mainPage.getByRole('button', { name: 'Show Logs' });
+      await playExpect(expandLogsButton).toBeVisible();
+      await expandLogsButton.click();
+      const machineCreationStatusMessage: Locator =
+        podmanOnboardingPage.mainPage.getByLabel('Onboarding Status Message');
+      await playExpect(machineCreationStatusMessage).toBeVisible({ timeout: PODMAN_MACHINE_STARTUP_TIMEOUT });
+      await playExpect(machineCreationStatusMessage).toHaveText('Podman successfully setup');
+      await podmanOnboardingPage.nextStepButton.click();
+    });
+    describe('Podman machine operations', async () => {
+      let podmanMachineStatus: Locator;
```

Comment:
Why aren't these buttons defined in the page object?

Reply:
I can extract them to a separate machine details page object.

Reply:
done.